CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#29](https://github.com/rubocop-hq/rubocop-minitest/pull/29): Add new `Minitest/RefuteRespondTo` cop.  ([@herwinw][])
```

Comment:
@herwinw : Can you add your GitHub profile URL here(https://github.com/rubocop-hq/rubocop-minitest/pull/29/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL44)? Your username should be clickable in the changelog.

Reply:
Fixed that as well.